R/load_data.R

```diff
@@ -113,14 +107,12 @@ ReadMPX_counts <- function (
 #' @examples
 #'
 #' library(pixelatorR)
-#' # Set arrow data output directory to temp for tests
-#' options(pixelatorR.arrow_outdir = tempdir())
 #'
 #' # Load example data as a Seurat object
 #' pxl_file <- system.file("extdata/five_cells",
 #'                         "five_cells.pxl",
 #'                         package = "pixelatorR")
-#' seur_obj <- ReadMPX_Seurat(pxl_file, overwrite = TRUE)
+#' seur_obj <- ReadMPX_Seurat(pxl_file)
```

Comment:
I get a warning `Warning: Data is of class matrix. Coercing to dgCMatrix.` for every call to ReadMPX_Seurat. No big deal, but perhaps people will get worried they are doing something wrong?

Reply:
Yeah I have noticed that too. I can add a conversion step to make sure that the matrix is always of class `dgCMatrix` before running `CreateAssayObject`.